Atari-Game-Player/steps/train.py

```diff
@@ -0,0 +1,170 @@
+from .configs import PreTrainingConfigs
+from DQN.model import *
+from zenml.steps import step, Output
+
+
+@step

```

Comment:
This function is too long. Maybe we can break it up into dfferent functions?

Reply:
with the function beeing this long it also becomes hard to remember that we are in one giant try except wrapper - moving some of this into sub-functions would make it a bit more understandable